tests/nexus/test_version.py

```diff
@@ -0,0 +1,14 @@
+"""
+Tests the version retrieval for the nexus definitions submodule
+"""
+from pynxtools import get_nexus_version
+
+
+def test_get_nexus_version():
+    """
+    Tests if we get a version string from nexus definitions
+    """
+    version = get_nexus_version()
+
+    assert version is not None
```

Comment:
should not we test something more specific, like to format of the string returned?

Reply:
Yes, I agree, but actually I was rather thinking about a regexp check that the format is according expectation

Reply:
I added this